README.md

```diff
@@ -56,8 +56,22 @@ celcl9aa-c3l7-7504-ca19-0c985e34ll8d
 In the Alfred worfklow in the upper right corner click the `[x]` icon and add the values from above to the corresponding value field
 
 [![img](https://i.imgur.com/Pe6nwey.jpg)](https://i.imgur.com/Pe6nwey.jpg)
+<!-- 
+JS: Remove this because notion_user_id is actually required in the code and this won't work.
 
-I recommend using chrome to retrieve these values. If you can only use safari you can copy the 'token_v2' value by following the equivalent steps above and populating the cookie env variable in Alfred so it looks like this `token_v2=XXXXXXXXXXXX`.
+I recommend using chrome to retrieve these values. If you can only use safari you can copy the 'token_v2' value by following the equivalent steps above and populating the cookie env variable in Alfred so it looks like this `token_v2=XXXXXXXXXXXX`. 
+
+-->
+
+## Troubleshooting
+
+It's possible (even likely) the script will fail due to an SSL error.  If the script isn't working, turn on debugging by clicking on the little cockroach in the alfred workflow screen.  If you see an error like:
```

Comment:
Nitpick but I'd remove: 'it's possible (even likely) ' 

and start with 'The script may fail...

We don't know how many folk experience the issue.